<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterMachine</class>
 <widget class="QDialog" name="AjouterMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter une machine</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>751</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1061</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/icons/Inter.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ajouter une machine</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Id Machine:</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_IDMACHINE</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Etat:</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_EtatM</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_Description_Ajout</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Ajouter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IDMACHINE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_EtatM">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Fonctionnel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>En panne</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Description_Ajout">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_IDSERVICE">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Id Service:</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_IDSERVICE</cstring>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_IDMACHINE</tabstop>
  <tabstop>comboBox_EtatM</tabstop>
  <tabstop>lineEdit_Description_Ajout</tabstop>
  <tabstop>comboBox_IDSERVICE</tabstop>
  <tabstop>pushButton_Ajouter</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
